### DISCUSSION TITLE
Does NTSC-RS support GPU acceleration?

### DISCUSSION THREAD

**Discussion by TheSystemGuy1337** [ntsc-rs/ntsc-rs#246] [Q&A]
I recently obtained a new PC with a RTX 4060 and I could see this program taking advantage of it (my previous PC had a horrible iGPU)

I'm wondering if NTSC-RS has GPU accelerated video rendering.

**Comment by valadaptive** [comment]
ntsc-rs runs entirely on the CPU. When I was first developing it, I thought about trying to run it on the GPU, but there are many parts that don't really work well with shaders.
